lib/components/viewers/nearby/stop.tsx

```diff
@@ -209,6 +214,7 @@ const mapStateToProps = (state: AppReduxState) => {
   const { config } = state.otp
   return {
     homeTimezone: config.homeTimezone,
+    nearbyViewConfig: config?.nearbyView,
```

Comment:
Could you just pass `hideEmptyStops` as a boolean prop instead of the whole thing?

Reply:
Was preparing for future nearby view config items